src/Nethermind/Nethermind.Facade/Proxy/RpcResult.cs

```diff
@@ -0,0 +1,46 @@
+//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+namespace Nethermind.Facade.Proxy
+{
+    public class RpcResult<T>
+    {
```

Comment:
why not using the existing solution?

Reply:
`Nethermind.JsonRpc` already contains a reference to `Nethermind.Facade` and the Proxy models are just plain DTOs without any additional processing logic whatsoever.